tools/mock-service-host/README.md

```diff
@@ -42,31 +37,28 @@ Stateless:
 
 -   Mock Service Host will return a valid mocked response for each valid call of GET/List.
 
-# Getting Started (with Azure CLI)
+# Getting Started
 
 ## Prerequisites
 
--   The [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli) is well installed.
--   The [Nodejs](https://nodejs.org/en/download/) is well installed.
-
-## Step 1: Start a Mock Service Host
+- [Nodejs](https://nodejs.org/en/download/) 14.15 or above
 
-### 1.1 Clone this project
+## Start Mock Service Host
 
-Use git to clone this repo to your local workspace.
+### Install Mock Server Host
 
+You need to create a workspace for the config and cache of Mock Server Host and then use `npm install` to get the latest Mock Server Host.
```

Comment:
Mock Service Host.
Could you do a full replacement in this spec to make the name consistent?